File: daemon/Models/Database/Journey/InformationType.cs
using NpgsqlTypes;

namespace daemon.Models.Database.Journey;

public enum InformationType
{
    [PgName("DISRUPTION")]
    DISRUPTION,
    [PgName("JOURNEY_ATTRIBUTE")]
    JOURNEY_ATTRIBUTE,
    [PgName("RIS_QUALITY_DEVIATION")]
    RIS_QUALITY_DEVIATION,
    [PgName("RIS_CAUSE_REASON")]
    RIS_CAUSE_REASON,
    [PgName("MESSAGE")]
    MESSAGE
}

File: frontend/src/lib/models/InformationType.ts
/**
 *
 * @export
 */
export const InformationType = {
	JourneyAttribute: "JOURNEY_ATTRIBUTE",
	Disruption: "DISRUPTION",
	Message: "MESSAGE",
	RisQualityDeviation: "RIS_QUALITY_DEVIATION",
	RisCauseReason: "RIS_CAUSE_REASON"
} as const;
export type InformationType = (typeof InformationType)[keyof typeof InformationType];
